person.rb

```diff
@@ -0,0 +1,19 @@
+# This is the Person class, parent of student and teacher class
+class Person
+  def initialize(age, id = 0, name = 'Unknown', parent_permission: true)
+    @id = id
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :age, :name
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] Let's make sure the `of_age?` method is a private method as said in the project description 👇🏻 

![Screenshot from 2022-10-05 02-30-12](https://user-images.githubusercontent.com/79058364/193954739-ad4a0f4f-a874-40a5-acdf-e92e716fe838.png)

Reply:
Thank you